docs/src/css/custom.css

```
@@ -200,80 +371,144 @@ a {
-[data-theme='dark'] ::-webkit-scrollbar {
-  width: 8px;
-  height: 8px;
+/* Footer. docusaurus.config.ts uses footer.style "light" so the color follows
+   the color mode instead of Infima's fixed slate: a quiet tinted band on light,
+   one step deeper than the page ground on dark. */
+.footer {
+  --ifm-footer-background-color: #f2f6fc;
+  --ifm-footer-color: var(--obot-slate-900);
+  --ifm-footer-link-color: var(--obot-slate-900);
+  --ifm-footer-title-color: var(--ifm-heading-color);
+  border-top: 1px solid var(--obot-border);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)